src/VisualStudio/Core/Test/Options/CSharpEditorConfigGeneratorTests.vb

```diff
@@ -83,6 +83,9 @@ dotnet_style_readonly_field = true:suggestion
 # Parameter preferences
 dotnet_code_quality_unused_parameters = all:suggestion
 
+# Suppression preferences
+dotnet_remove_unnecessary_suppression_exclusions = none
```

Comment:
exclusions?  what is this for?

Reply:
This allows to turn off this analyzer for specific diagnostic IDs. I found this could be a common requirement for shared projects where say specific diagnostics are enabled in some projects and disabled in other. It would help to allow just turning off analysis for suppressions of these IDs.